src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,5 @@
+$card-height: 300px;
+$card-width: 200px;
+$transition-duration: 300ms;
+$title-hover-color: #34568b;
+$link-hover-color: #00acdc;
```

Comment:
look at the last item in the checklist - use variables for the main values so that you'll be able to reuse them and give them descriptive names. But don't overuse them, don't create variable for the value that's used just once.
maybe its better not to use these variables (except transition one)

Reply:
Totally agree. Overused. Fixed